js/src/forum/components/DiscussionPage.js

```diff
@@ -199,6 +200,11 @@ export default class DiscussionPage extends Page {
     this.stream = new PostStream({ discussion, includedPosts });
     this.stream.on('positionChanged', this.positionChanged.bind(this));
     this.stream.goToNumber(m.route.param('near') || (includedPosts[0] && includedPosts[0].number()), true);
+
+    app.current.setData({
+      discussion: discussion,
+      stream: this.stream,
+    });
```

Comment:
What do these calls do? They don't seem to replicate any previously existing logic...

Reply:
There's plenty of references to app.current.stream throughout. That was previously available by default, but not now.